gen-apidocs/generators/api/definition.go

```diff
@@ -224,6 +224,10 @@ func (d *Definition) GetOperationGroupName() string {
 	if strings.ToLower(d.Group.String()) == "rbac" {
 		return "RbacAuthorization"
 	}
+	if strings.ToLower(d.Group.String()) == "flowcontrol" {
+		return "FlowcontrolApiserver"
```

Comment:
@tengqm , why are `flowcontrol` and `rbac` special cases? How is the return string determined?

Reply:
Oh, I missed the detail in your commit note. OK.